src/patternfly/components/JumpLinks/jump-links.scss

```
@@ -62,8 +62,9 @@ $pf-v6-c-jump-links--m-expandable--breakpoint-map: build-breakpoint-map("base",
 
   // toggle icon
   --#{$jump-links}__toggle-icon--Color: var(--pf-t--global--icon--color--regular);
-  --#{$jump-links}__toggle-icon--TransitionDuration: var(--pf-t--global--motion--duration--icon--long);
+  --#{$jump-links}__toggle-icon--TransitionDuration: var(--pf-t--global--motion--duration--icon--default);
   --#{$jump-links}__toggle-icon--TransitionTimingFunction: var(--pf-t--global--motion--timing-function--default);
+  --#{$jump-links}__toggle-icon--Transition: tranform var(--#{$jump-links}__toggle-icon--TransitionDuration) var(--#{$jump-links}__toggle-icon--TransitionTimingFunction); // TODO remove in breaking change along with shorthand property
```

Typo in transform - but you can remove this `--Transition` var and make it longhand, too.

```suggestion
  --#{$jump-links}__toggle-icon--Transition: transform var(--#{$jump-links}__toggle-icon--TransitionDuration) var(--#{$jump-links}__toggle-icon--TransitionTimingFunction); // TODO remove in breaking change along with shorthand property
```